doc/user_doc/vic_app_dev/container_operations.md

```diff
@@ -29,12 +29,12 @@
 |`build`|[Build an image from a Dockerfile](https://docs.docker.com/engine/reference/commandline/build/)|No|
 |`commit`|[Create a new image from a container’s changes](https://docs.docker.com/engine/reference/commandline/commit/)|No|
 |`history`|[Show the history of an image](https://docs.docker.com/engine/reference/commandline/history/)|No|
-|`images`|[Images](https://docs.docker.com/engine/reference/commandline/images/)|Yes. vSphere Integrated Container Engine currently supports `--no-trunc` and `-q/--quiet`|
+|`images`|[Images](https://docs.docker.com/engine/reference/commandline/images/)|Yes. Supports `--filter`, `--no-trunc`, and `--quiet`|
 |`import`|[Import the contents from a tarball to create a filesystem image](https://docs.docker.com/engine/reference/commandline/import/)|No|
 |`load`|[Load an image from a tar archive or STDIN](https://docs.docker.com/engine/reference/commandline/load/)|No|
-|`rmi`|[Remove a Docker image](https://docs.docker.com/engine/reference/commandline/rmi/)|Yes. vSphere Integrated Container Engine does not yet support tag or untag operations, or any options.|
+|`rmi`|[Remove a Docker image](https://docs.docker.com/engine/reference/commandline/rmi/)|Yes. Does not yet support `tag` or `untag` operations, or any options.|
```

Comment:
It seems not: https://docs.docker.com/engine/reference/commandline/rmi/

I can only imagine that this is a copy/paste error. Removed.